### PR TITLE
Move FIPS and CC-EAL2 and CIS under under a Certifications column

### LINKED ISSUE #6154: /advantage invites people to use non-functional FIPS/CC-EAL
From @BryanQuigley in #5953:
> For 14.04 launch, I think FIPS/CC should just say Coming Soon! See https://docs.ubuntu.com/security-certs/en/ for current instructions.

And in #6136:
> FIPS and CC-EAL don't do anything today, we should not encourage anyone to run those commands

### LINKED ISSUE #6710: /advantage: Add “CIS benchmark” (cis-audit)
1\. Go to [/advantage](https://ubuntu.com/advantage)
2\. Look for info about the CIS benchmark
3\. Sign in
4\. Look for instructions about how to turn the CIS service on/off

What happens: 2, 4. There is no such info.

What should happen: CIS is covered in all the same places that FIPS and CC-EAL are.

This will involve finding out what exactly happens when you “enable” `cis-audit`. Presumably it doesn’t just download the benchmarks PDF. So what does it do?

[Based on [a comment by @lechsandecki in the copydoc](https://docs.google.com/document/d/1AW8wQu1Uvq0xFtd_c2JwNdic075MPOTx7k2KHqFPPeo/edit?disco=AAAAGN2UYVE).]

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done
Move FIPS and CC-EAL2 and CIS under under a Certifications column

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Make sure there is a `Certifications` column on the contracts table and that when you expand it the content matches the [specification document](https://docs.google.com/document/d/1K65n3NpZCh7EWDXKLhG31seGJ2jKbW6lqw3BwV2mlnU/), including that we are linking to the right place

## Issue / Card
Fixes #6154
Fixes #6710

### CODE REVIEW

File: templates/advantage/index.html [hunk 1]
```diff
@@ -307,31 +296,28 @@ <h2>Your paid subscriptions</h2>
...
+                            <ul class="p-list" style="padding-left: 2rem;">
+                              <li>Available on: Ubuntu 16.04 LTS</li>
+                              <li>To activate CC-EAL2: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cc-16/">CC-EAL2 setup instructions</a></li>
+                            </ul>
+                            
+                            <p><strong>CIS tools</strong> test compliance with the CIS security benchmark.</p>
+                            <ul class="p-list" style="padding-left: 2rem;">
+                              <li>Available on: Ubuntu 16.04 LTS, 18.04 LTS</li>
+                              <li>To use the tools: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cis-18-16.html">CIS setup instructions</a></li>
+                            </ul>
```
Comment (sowasred2012) [inline templates/advantage/index.html:320]: Alignment was looking a little weird - but if we do this (based off what's in the "Infra" tab), it looks neater:
```suggestion
                          <div class="col-12 u-align--center">
                            <strong>FIPS 140-2</strong> Level 1 certified packages use older software versions.<br />
                            Use them only if your organisation requires it.<br /><br />
                            
                            Available on: Ubuntu 16.04 LTS, 18.04 LTS<br />
                            To activate FIPS: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/fips">FIPS setup instructions</a><br /><br />
                            
                            <strong>CC-EAL2</strong> certified packages use older software versions<br />
                            Use them only if your organisation requires it.<br /><br />
                            
                            Available on: Ubuntu 16.04 LTS<br />
                            To activate CC-EAL2: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cc-16/">CC-EAL2 setup instructions</a><br /><br />
                            
                            <strong>CIS tools</strong> test compliance with the CIS security benchmark.<br /><br />

                            Available on: Ubuntu 16.04 LTS, 18.04 LTS<br />
                            To use the tools: <a class="p-link--external" href="https://docs.ubuntu.com/security-certs/en/cis-18-16.html">CIS setup instructions</a>
```

**Comment by jpmartinspt** [comment]
@sowasred2012 accepted the suggestions, looks better indeed.

**Comment by jpmartinspt** [comment]
@meralonso does this look alright?
imagem

**Comment by meralonso** [comment]
@jpmartinspt how do the other cells look? Is the text always center aligned?

**Comment by jpmartinspt** [comment]
@meralonso it is, but generally it's less text.

imagem

**Comment by meralonso** [comment]
@jpmartinspt personally, I'd go left align as I feel it should be a good reason to go center align, it just makes things a bit more complicated to read. But if that's the way text is laid out in all cells then let's leave it as it is.

**Comment by jpmartinspt** [comment]
@meralonso better?
imagem

**Comment by meralonso** [comment]
yes, looking better

**Pull request merged**